### PR TITLE
Dts

### PR DESCRIPTION
**Pull request by erlendcognite**

### CODE REVIEW

**Comment by cognite-cicd** [comment]
[Tests failed](https://jenkins.cognite.ai/job/github-builds/job/cognite-sdk-python/job/PR-126/1/testReport/).

#### TestDatapoints.test_get_datapoints[get_dps_response_obj0]
```
@pytest.fixture(scope="class")
    def datapoints_fixture():
        tso = dto.TimeSeries(TS_NAME)
>       time_series.post_time_series([tso])
E       AttributeError: module 'cognite.v06.time_series' has no attribute 'post_time_series'

tests/v06/test_datapoints.py:28: AttributeError
```

#### TestDatapoints.test_get_dps_output_formats[get_dps_response_obj0]
```
@pytest.fixture(scope="class")
    def datapoints_fixture():
        tso = dto.TimeSeries(TS_NAME)
>       time_series.post_time_series([tso])
E       AttributeError: module 'cognite.v06.time_series' has no attribute 'post_time_series'

tests/v06/test_datapoints.py:28: AttributeError
```

#### TestDatapoints.test_get_dps_correctly_spaced[get_dps_response_obj0]
```
@pytest.fixture(scope="class")
    def datapoints_fixture():
        tso = dto.TimeSeries(TS_NAME)
>       time_series.post_time_series([tso])
E       AttributeError: module 'cognite.v06.time_series' has no attribute 'post_time_series'

tests/v06/test_datapoints.py:28: AttributeError
```

... and 7 more test failures. Go to Jenkins for the full list.

**Comment by erlendcognite** [comment]
Forgot to include in commit message: Also added support in DataSpec and DataTransfer service for labeling timeseries for resulting dataframe.

File: cognite/v05/dto.py [hunk 1]
```diff
@@ -266,15 +266,15 @@ def to_pandas(self):
...
-        isString (bool):    Whether the time series is string valued or not.
+        is_string (bool):    Whether the time series is string valued or not.
         metadata (dict):    Metadata.
         unit (str):         Physical unit of the time series.
-        assetId (str):     Asset that this time series belongs to.
+        asset_id (str):     Asset that this time series belongs to.
         description (str):  Description of the time series.
-        securityCategories (list(int)): Security categories required in order to access this time series.
-        isStep (bool):        Whether or not the time series is a step series.
+        security_ategories (list(int)): Security categories required in order to access this time series.
```
Comment (NilsBarlaug-cognite) [inline cognite/v05/dto.py]: security_ategories

File: cognite/v06/datapoints.py [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+from cognite.v05.dto import DatapointsResponse
+
+
+def get_datapoints(id, aggregates=None, granularity=None, start=None, end=None, **kwargs):
+    """Returns a DatapointsObject containing a list of datapoints for the given query.
+
+    This method will automate paging for the user and return all data for the given time period.
+
+    Args:
+        id (int):             The unique id of the timeseries to retrieve data for.
```
Comment (NilsBarlaug-cognite) [inline cognite/v06/datapoints.py]: Does it matter that the indentation is off?

File: cognite/v06/datapoints.py [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+
+        api_key (str):          Your api-key.
+
+        project (str):          Project name.
+
+        limit (str):            Max number of datapoints to return. If limit is specified, this method will not automate
+                                paging and will return a maximum of 100,000 dps.
+
+    Returns:
+        v05.dto.DatapointsResponse: A data object containing the requested data with several getter methods with different
```
Comment (NilsBarlaug-cognite) [inline cognite/v06/datapoints.py]: Should 0.6 return something from 0.5?

File: cognite/v06/time_series.py [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+        include_metadata (bool):    Decide if the metadata field should be returned or not. Defaults to False.
+
+    Keyword Arguments:
+        api_key (str):          Your api-key.
+
+        project (str):          Project name.
+
+    Returns:
+        v05.dto.TimeSeriesResponse: A data object containing the requested timeseries.
```
Comment (NilsBarlaug-cognite) [inline cognite/v06/time_series.py]: Same question?

**Comment by cognite-cicd** [comment]
[Tests failed](https://jenkins.cognite.ai/job/github-builds/job/cognite-sdk-python/job/PR-126/5/testReport/).

#### TestDataTransferService.test_instantiate_data_spec
```
@pytest.fixture
    def ts_data_spec_dtos():
        time_series = [
>           TimeSeries(name="constant", aggregates=["step"], missing_data_strategy="ffill"),
            TimeSeries(name="sinus"),
        ]
E       TypeError: __init__() got an unexpected keyword argument 'name'

tests/test_data_transfer_service.py:21: TypeError
```

#### TestDataTransferService.test_instantiate_ts_data_spec_time_series_not_list
```
self = <tests.test_data_transfer_service.TestDataTransferService object at 0x7fa18b40fe10>

    def test_instantiate_ts_data_spec_time_series_not_list(self):
        with pytest.raises(DataSpecValidationError):
            DataSpec(
                time_series_data_specs=[
>                   TimeSeriesDataSpec(time_series=TimeSeries(name="ts1"), aggregates=["avg"], granularity="1s")
                ]
            )
E           TypeError: __init__() got an unexpected keyword argument 'name'

tests/test_data_transfer_service.py:105: TypeError
```

#### TestDataTransferService.test_json_dumps_loads
```
@pytest.fixture
    def ts_data_spec_dtos():
        time_series = [
>           TimeSeries(name="constant", aggregates=["step"], missing_data_strategy="ffill"),
            TimeSeries(name="sinus"),
        ]
E       TypeError: __init__() got an unexpected keyword argument 'name'

tests/test_data_transfer_service.py:21: TypeError
```

... and 5 more test failures. Go to Jenkins for the full list.

**Pull request merged**